client-reactjs/src/components/layout/Header.js

```diff
@@ -70,13 +70,18 @@ class AppHeader extends Component {
     }
 
     componentDidMount(){
+      if(window.location.href.split("/").includes('manualJobDetails')){
```

Comment:
Please check if react router can be used here to check the path

Reply:
User React router

Reply:
User React router